### ISSUE TITLE
Node-gyp install failure on macOS

### ISSUE THREAD

**Issue by JonasKruckenberg** [kriszyp/cbor-extract#1]
When installing cbor-extract for cbor-x I've run into the following error:
```
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.0.0 | darwin | x64
gyp info find Python using Python version 2.7.16 found at "/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python"
(node:6061) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
gyp info spawn args [
gyp info spawn args   '/usr/local/Cellar/node/16.0.0/libexec/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '[...]node_modules/cbor-extract/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/usr/local/Cellar/node/16.0.0/libexec/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '[...]Library/Caches/node-gyp/16.0.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=[...]Library/Caches/node-gyp/16.0.0',
gyp info spawn args   '-Dnode_gyp_dir=/usr/local/Cellar/node/16.0.0/libexec/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=[...]Library/Caches/node-gyp/16.0.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=[...]node_modules/cbor-extract',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/cbor-extract/src/extract.o
In file included from ../src/extract.cpp:8:
In file included from [...]Library/Caches/node-gyp/16.0.0/include/node/v8.h:30:
[...]Library/Caches/node-gyp/16.0.0/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/type_traits:697:50: note: 'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
../src/extract.cpp:170:17: warning: unused variable 'context' [-Wunused-variable]
        Local<Context> context = Nan::GetCurrentContext();
                       ^
1 warning and 1 error generated.
make: *** [Release/obj.target/cbor-extract/src/extract.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/Cellar/node/16.0.0/libexec/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:365:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Darwin 20.3.0
gyp ERR! command "/usr/local/Cellar/node/16.0.0/bin/node" "/usr/local/Cellar/node/16.0.0/libexec/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd [...]node_modules/cbor-extract
gyp ERR! node -v v16.0.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok
```

System:
- macOS: 11.2.3 (Big Sur)
- node: v16.0.0
- cbor-extract: v0.2.0
- node-gyp: v7.1.2

Hope this helps to resolve the issue, cbor-x is faster than every other encoder out there without, but I'm excited to see how fast it can get!

**Comment by kriszyp** [comment]
Should be fixed in v0.2.1 (and corresponding cbor-x v0.6.4). Thanks for the report.

**Comment by JonasKruckenberg** [comment]
This issue is resolved so no need to keep this around 👍🏻 thanks for your effort!
